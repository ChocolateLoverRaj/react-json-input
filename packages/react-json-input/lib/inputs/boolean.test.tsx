import { render } from '@testing-library/react'
import { JsonInput } from '..'

test('readOnly', () => {
  expect(render(
    <JsonInput schema={{ type: 'boolean' }} readOnly />
  ).baseElement).toMatchSnapshot()
})
