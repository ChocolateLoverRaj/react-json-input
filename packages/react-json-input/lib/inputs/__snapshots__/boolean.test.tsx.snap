// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`readOnly 1`] = `
<body>
  <div>
    <table>
      <tbody>
        <tr>
          <td>
            <span>
              âœ“
            </span>
          </td>
          <th
            class="react-json-input__lib_InputName.module.scss__inputName"
          />
          <td>
            <input
              disabled=""
              type="checkbox"
            />
          </td>
          <td>
            <select
              disabled=""
            >
              <option
                value="0"
              >
                boolean
              </option>
            </select>
          </td>
          <td />
        </tr>
      </tbody>
    </table>
  </div>
</body>
`;
